pyccel/parser/semantic.py

```diff
@@ -443,15 +443,20 @@ def get_function(self, name):
 
 
     def get_import(self, name):
-        """."""
+        """
+        Search for an import with the given name in the current namespace.
+        Return None if not found.
+        """
 
         imp = None
-
         container = self.namespace
         while container:
 
-            if name in container.imports['imports']:
-                imp =  container.imports['imports'][name]
+            for key,_ in container.imports['imports'].items():
```

Comment:
If you don't need the values of a dictionary you can use the `.keys()` method instead of the `.items()` method